core/exec-operand.c

```diff
@@ -78,7 +79,16 @@ void lai_exec_access(lai_variable_t *object, lai_nsnode_t *src) {
         case LAI_NAMESPACE_INDEXFIELD:
             lai_read_opregion(object, src);
             break;
-        case LAI_NAMESPACE_DEVICE:
+		case LAI_NAMESPACE_BUFFER_FIELD:
+			lai_read_buffer(object, src);
+			break;
+		case LAI_NAMESPACE_EVENT:
```

Comment:
Maybe add a `__attribute__ ((fallthrough))` or `/* fall through */` to indicate that the fallthrough is intended